bot/seasons/easter/easter_riddle.py

```diff
@@ -0,0 +1,99 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'easter_riddle.json'), 'r', encoding="utf8") as f:
```

Comment:
In an earlier PR (6ccaa6400f1e7683598422c3a80ee2eb8ad108ca) someone went over all the paths and made them into one string where possible. This will be more consistent with that. Also, let's use `Path.open()`.

```suggestion
with Path("bot/resources/easter/easter_riddle.json").open("r", encoding="utf8") as f:
```